### PR TITLE
[wip] refactor url in router

### PR DESCRIPTION
**Pull request by lhy1024**
### What problem does this PR solve? 

There are strings about url in two package, pd-ctl and router. This PR will merge them.

### Check List 

Tests 

 - Unit test

### CODE REVIEW

File: server/api/router.go [hunk 1]
```diff
@@ -22,7 +22,110 @@ import (
...
+	//DeprecatedHealths is the api that may GET.
+	DeprecatedHealths = "/health"
+	//DeprecatedDiagnose is the api that may GET.
+	DeprecatedDiagnose = "/diagnose"
+	Classifier         = "/api/v1/classifier/"
+	NameSpace          = "/table/namespaces"
+	NameSpaceTable     = "/table/namespaces/table"
+	NameSpaceMeta      = "/table/namespaces/meta"
+	StoreNameSpace     = "/table/store_ns"
+)
```
Comment (disksing) [inline server/api/router.go]: I think it would be better to move comments to the end of line to make the layout more tidy.

Comment (disksing) [reply server/api/router.go]: the rest LGTM.

Comment (lhy1024) [reply server/api/router.go]: I have try to add comment `Ping = "/ping" // Ping is the api that may GET.`, but it can not pass the go lint test,

Comment (disksing) [reply server/api/router.go]: 😞 maybe we don't have to export all paths.

**Pull request closed without merging**